src/middlewared/middlewared/plugins/zfs.py

```diff
@@ -916,6 +918,29 @@ def cancel(self):
         self._cancel.set()
 
 
+class ScrubNotStartedAlertClass(AlertClass, OneShotAlertClass):
+    category = AlertCategory.TASKS
+    level = AlertLevel.WARNING
+    title = "Scrub Not Started"
```

Comment:
Scrub Not Started seems strange, perhaps only use it for actual failures and say "Scrub Failed to Start" or something?

Then do not generate it when scrub or resilver is in progress.

Reply:
Do we drop the notification that scrub was not started due to another scrub/resilver?